src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -277,6 +272,10 @@ public async Task InitializeAsync()
                 // Generate Functions
                 IEnumerable<FunctionMetadata> functions = GetFunctionsMetadata();
 
+                _workerRuntime = _workerRuntime ?? Utility.GetWorkerRuntime(functions);
```

Comment:
Is the call to `GetWorkerRuntime()` the fix? I assume we were logging "none" a lot?

Reply:
Yes. GetWorkerRuntime calculates language for an app if app setting "Functions_Worker_Runtime" is not set.